src/main.cpp

```diff
@@ -2,136 +2,114 @@
 #include <MFRC522.h>
 #include "SoftwareSerial.h"
 #include "DFRobotDFPlayerMini.h"
+#include <FastLED.h>
 
 // logic
 #define MY_LEVEL 1
 
 // RFID
 #define SS_PIN 10
 #define RST_PIN 9
-MFRC522 rfid(SS_PIN, RST_PIN); // Instance of the class
+MFRC522 mfrc522(SS_PIN, RST_PIN); // Instance of the class
+#include "MFRC522_func.h"
```

Comment:
`#include`s are ususally placed together at the beginning of the file.
You can move this to follow `#include <MFRC522.h>`